packages/babel-core/src/config/files/configuration.js

```diff
@@ -2,6 +2,7 @@
 
 import buildDebug from "debug";
 import path from "path";
+import codeFrame from "babel-code-frame";
```

Comment:
Should this should be `@babel/code-frame`?

Reply:
yes thanks